packages/components/src/components/Table/Table.tsx

```diff
@@ -30,30 +38,36 @@ const ScrollContainer = styled.div`
 export type TableProps = AllowedFrameProps & {
     children: ReactNode;
     colWidths?: string[];
+    highlightRowOnHover?: boolean;
 };
 
-export const Table = ({ children, margin, colWidths }: TableProps) => {
+export const Table = ({ children, margin, colWidths, highlightRowOnHover = false }: TableProps) => {
```

Comment:
Please add `highlightRowOnHover` to storybook

Reply:
Done